tools/docker/tests/test_gw_repeater.sh

```diff
@@ -66,7 +66,7 @@ main() {
             * ) err "unsupported argument $1"; usage; exit 1 ;;
         esac
     done
-
+  
```

Comment:
Whitespace error introduced here.

Reply:
Will fix all whitespace errors in this file while doing so, is that ok?